roles/unicorn_user/tasks/main.yml

```diff
@@ -33,3 +33,10 @@
     owner: "{{ unicorn_user }}"
     group: "{{ unicorn_user }}"
     mode: 0644
+
+- name: Create sudoers configuration for unicorn user
+  become: true
+  template:
+    src: sudoers.j2
+    dest: "/etc/sudoers.d/90-unicorn"
```

Comment:
I'd add a different number other than the one in the existing sudoers file to make a bit more obvious what the actual order is. According to the man page, having numbers repeated in filenames doesn't cause any problem though.